ci/pipelines/control-plane.yml

```diff
@@ -10,6 +10,24 @@ resources:
       username: ((cf-username))
       password: ((cf-password))
 
+  - name: pay-control-plane-src
+    type: git
+    icon: github-circle
```

Comment:
think this needs to just be github. It's showing just a question mark atm. It has to be an icon available in https://materialdesignicons.com/

Reply:
Aha - https://github.com/alphagov/pay-team-manual/blob/master/ci/pipeline/pay-team-manual.yml#L11

Reply:
Oh yeah, that one is also broken 😆
<img width="580" alt="Screenshot 2022-03-21 at 16 06 34" src="https://user-images.githubusercontent.com/2170030/159302387-3d511cb3-4ac6-4550-a88a-7bfa028d6180.png">